components/player-view.js

```diff
@@ -1,318 +1,137 @@
 const React = require('react');
-const cn = require('classnames');
 const deepAssign = require('deep-assign');
+const emitter = require('../client-lib/emitter');
+const PlayerControls = require('../components/player-controls');
+const ReplayView = require('../components/replay-view');
+const Progress = require('../components/progress');
+
+module.exports = class Player extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {hovered: false};
```

Comment:
This seems to occur everywhere. Any reason it couldn't be pulled up into a parent base class?